src/StakingNode.sol

```diff
@@ -461,50 +559,21 @@ contract StakingNode is IStakingNode, StakingNodeEvents, ReentrancyGuardUpgradea
      * @return True if the delegation state is synced, false otherwise.
      */
     function isSynchronized() public view returns (bool) {
-        IDelegationManager delegationManager = IDelegationManager(address(stakingNodesManager.delegationManager()));
+        IDelegationManager delegationManager = stakingNodesManager.delegationManager();
         return delegatedTo == delegationManager.delegatedTo(address(this));
     }
 
-    function synchronize(uint256 queuedShares, uint32 undelegateBlockNumber) public onlyDelegator {
```

Comment:
great to see this whole thing gone.

It was useful while it lasted since that's how the old eigenlayer worked but glad to see we can now just sync queued shares